src/Shared/test/Shared.Tests/runtime/Http2/HPackDecoderTest.cs

```diff
@@ -44,6 +44,8 @@ public class HPackDecoderTests
 
         private const string _userAgentString = "user-agent";
 
+        private static readonly byte[] _userAgentBytes = Encoding.ASCII.GetBytes(_userAgentString);
```

Comment:
This was removed by https://github.com/dotnet/aspnetcore/commit/af9bb41d623b17eec946029815db5554b73be156#diff-f667c36e873f2a57f94e59f6748e4f3f4318ccbc55f67a51bf1e09616a809ce5, I'll update runtime.